katsdpimager/report.py

```diff
@@ -172,13 +198,21 @@ def __init__(self, dataset: katdal.DataSet,
         mask = katsdpimager.metadata.target_mask(dataset, target)
         self.timestamps = dataset.timestamps[mask]
         self.time_range = bokeh.models.Range1d(
-            datetime.fromtimestamp(self.timestamps[0], timezone.utc),
-            datetime.fromtimestamp(self.timestamps[-1], timezone.utc)
+            datetime.fromtimestamp(self.timestamps[0] - 0.5 * dataset.dump_period, timezone.utc),
```

Comment:
There is also `dataset.start_time` and `dataset.end_time`...

Reply:
But that's for the whole dataset, rather than for the time spent on the target (self.timestamps is a subset of dataset.timestamps).

Reply:
Ah, yes, good point.